src/db/models/request.js

```diff
@@ -1,24 +1,22 @@
-const connection = require('../connection');
-const {Sequelize} = connection;
-
-const Request = connection.define('request', {
-    id: {
-        type: Sequelize.INTEGER,
-        primaryKey: true,
-        autoIncrement: true
-    },
-    requesterUid: {
+module.exports = (sequelize) => {
```

Comment:
Rather than going up the hierarchy to get an instance of Sequelize, it's better to just return a function that the main connection class can pass its own instance in.